native/src/window/action.rs

```diff
@@ -35,6 +35,17 @@ pub enum Action<T> {
     SetMode(Mode),
     /// Sets the window to maximized or back
     ToggleMaximize,
+    /// Brings the window to the front and sets input focus. Has no effect if the window is
+    /// already in focus, minimized, or not visible.
+    ///
+    /// This method steals input focus from other applications. Do not use this method unless
+    /// you are certain that's what the user wants. Focus stealing can cause an extremely disruptive
+    /// user experience.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / Wayland:** Unsupported.
```

Comment:
No need to mention `iOS / Android` since we do not officially support mobile platforms!